common/persistence/src/main/java/app/coronawarn/server/common/persistence/repository/CustomDiagnosisKeyRepository.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Corona-Warn-App
+ *
+ * SAP SE and all other contributors /
+ * copyright owners license this file to you under the Apache
+ * License, Version 2.0 (the "License"); you may not use this
+ * file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package app.coronawarn.server.common.persistence.repository;
+
+import java.util.List;
+
+public interface CustomDiagnosisKeyRepository<T> {
```

Comment:
Consider not using interfaces if only a single implementation exists. Or are further `CustomDiagnosisKeyRepository`s planned? Then the implementation should not carry the `Impl` suffix, but be more descriptive.